### DISCUSSION TITLE
Support for dragging items from one grid to another

### DISCUSSION THREAD

**Discussion by kevin0756** [react-grid-layout/react-grid-layout#2078] [Q&A]
I believe this was asked earlier in [792](https://github.com/react-grid-layout/react-grid-layout/issues/792) which was a duplicate of [688](https://github.com/react-grid-layout/react-grid-layout/issues/688). There was a PR created in issue 688 but this was not merged. I would like to know if there is a solution present right now for this to work. In issue 792 it also mentions an outdated fork called `react-grid-layout-between` for addressing this use case. However that repo is not updated since 2020 and it does not support react 17 and above (besides the obvious vulnerabilities).

Warm Regards,
Kevin Lewis

**Comment by luifermoron** [comment]
I found a workaround by implementing mousemove events and custom x,y calculations but it's not accurate enough. 

I was reviewing the code of the library and I think the internal states that needs to be modified are: droppingDOMNode and droppingPosition and may be onDragOver function as well. 

I can work with someone of the contributors to make it work. what do you think?

**Comment by sguisse** [comment]
Does the new version of RGL 2.1.0 support this functionality ?

**Reply by STRML** [reply]
It does not, out of the box. But perhaps it could be written as a plugin now.

**Comment by nhaebigkerber** [comment]
Have there been any updates on this in the new year? We were preparing to adopt this library at my organization but this is a dealbreaker for us.

**Reply by STRML** [reply]
Go ahead and build it!

**Reply by slacroix** [reply]
Hey @STRML ! Any guidance on where to start with plugins? We need it as well and we're struggling to upgrade from 1.x to 2.x
